service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Element.java

```diff
@@ -12,11 +13,18 @@
 
 @Data
 @Builder
-@AllArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Element<T> {
+
     private final UUID id;
     @NotNull
     @Valid
     private final T value;
+
+    @JsonCreator
+    public Element(@JsonProperty("id") final UUID id,
```

Comment:
Any value in bringing this back in instead of allArgsConstructor?

Reply:
👍